### ISSUE TITLE
HC-App: Make underpayments easier to manage

### ISSUE THREAD

**Issue by diabWH3** [James-A-White/HarrierCentral#250]
I had a hasher with not enough money for a previous run (Puget Sound H3 #1106 - run cost: $10.00) and couldn't figure out how to mark an underpayment at the time, due to the new custom "Other" payment options. I figured it out by entering a negative "top-up" amount, but this could be clearer. This is not a very common occurrence, but one that does happen every now and again.

![Screenshot_20220119-160020](https://user-images.githubusercontent.com/49632769/150238097-60c28d4a-6905-4b11-aa19-2b297f9f468e.png)

**Comment by James-A-White** [comment]
Just so I understand... the person came to a $10 run with only $8... you wanted to take the $8 and put the rest on credit. Correct?

**Comment by diabWH3** [comment]
Correct, they scrounged their wallet and only came up with $8, so $2 short.
With my work-around, they now show as owing the kennel $2.
